client/src/components/frontend/Resource/Detail.js

```diff
@@ -14,6 +14,13 @@ export default class ResourceDetail extends Component {
     resource: PropTypes.object
   };
 
+  createDescription(description) {
```

Comment:
We weren't actually inserting the HTML into the descriptions, so none of the formatting was being preserved.  Let me know if there is a different way we should be doing this.